rskj-core/src/test/java/co/rsk/net/light/LightProcessorTest.java

```diff
@@ -613,6 +586,32 @@ private static TransactionReceipt createReceipt() {
         return receipt;
     }
 
+    private Block mockBestBlock() {
+        Block bestBlock = mock(Block.class);
+        when(bestBlock.getNumber()).thenReturn((long) 50);
+        when(blockStore.getBestBlock()).thenReturn(bestBlock);
+        return bestBlock;
+    }
+
+    private Block createMockedBlockAndIncludeInBlockchain(long blockNumber, byte[] blockHash) {
+        final Block block = mock(Block.class);
+        final BlockHeader startBlockHeader = mock(BlockHeader.class);
+
```

Comment:
Move this methods to the LightClientTestUtils! That was why it was created for haha

Reply:
Yes, I know. but the branch lc-SyncRound which I'm working on has more interesting methods to move to LightClientTestUtils, so after merged, I'll move those methods to Utils.